### PR TITLE
Github Actions

### PR DESCRIPTION
**Pull request by BognaLew**

### CODE REVIEW

File: .github/workflows/linter.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          # Full git history is needed to get a proper
+          # list of changed files within `super-linter`
+          fetch-depth: 0
+      - name: Lint Code Base
+        uses: super-linter/super-linter@v6.3.0
+        env:
+          VALIDATE_ALL_CODEBASE: true
```
Comment (bstrzelecki) [inline .github/workflows/linter.yml:24]:
```suggestion
          VALIDATE_ALL_CODEBASE: true
          FILTER_REGEX_EXCLUDE: "/github/workspace/api_gateway/gradlew"
```
exclude auto-generated file

**Review by bstrzelecki — APPROVED** [review]

File: .github/workflows/linter.yml (hunk 1 above)
Comment (Jackenmen) [reply .github/workflows/linter.yml:24]: Let's not include the GH path in the filter:
```suggestion
          VALIDATE_ALL_CODEBASE: true
          FILTER_REGEX_EXCLUDE: "api_gateway/gradlew"
```

Comment (bstrzelecki) [reply .github/workflows/linter.yml:24]:
```suggestion
          VALIDATE_ALL_CODEBASE: true
          FILTER_REGEX_EXCLUDE: "*/api_gateway/gradlew"
```
Then it should be like this

File: .github/workflows/linter.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v4
+        with:
+          # Full git history is needed to get a proper
+          # list of changed files within `super-linter`
+          fetch-depth: 0
+      - name: Lint Code Base
+        uses: super-linter/super-linter@v6.3.0
```
Comment (Jackenmen) [inline .github/workflows/linter.yml:22]: Let's go with v6.3.1 as v6.3.0 has a bug that prevents it from running on my system and I want to be consistent with the version used in CI:
```suggestion
        uses: super-linter/super-linter@v6.3.1
```

File: .github/workflows/linter.yml (hunk 1 above)
Comment (Jackenmen) [reply .github/workflows/linter.yml:24]: That's not a valid regex, a dot is missing :)

Let's go with a simple:
```suggestion
          VALIDATE_ALL_CODEBASE: true
          FILTER_REGEX_EXCLUDE: ".*/gradlew"
```
We're going to need for all of the services anyway.

**Review by Jackenmen — APPROVED** [review]

**Review by bstrzelecki — APPROVED** [review]

**Pull request merged**